frontend/interfaces/UserTypes.ts

```diff
@@ -19,15 +21,53 @@ interface AbstractUserData {
   profilePicture: string;
 }
 
-export interface NewUserData extends AbstractUserData {}
+export interface PublicUserData {
```

Comment:
i wonder how we can type these classes to be linked with the `AbstractUserData` classes above, just so the linter and IDE's can pick up on duplicated fields for you and stuff liket that... 🤔

Reply:
I had a search through google for linting but didn't really find anything that would prevent duplication of property from happening 🤔